/// Expect:
/// - output: "PASS\n"

class Foo {
    public fn toss(this) throws {
        throw Error::from_errno(0)
    }
}

fn main() {
    let foo = Foo()
    try foo.toss() catch error {
        println("PASS")
    }
}
